### ISSUE TITLE
Cannot use quality sliders.

### ISSUE THREAD

**Issue by LapisWolfMC** [sp614x/optifine#2626]
So in Minecraft 1.14.3 with the new version just released, you cannot use any of the quality sliders. In 1.14, Mojang added a feature that when something is changed, it reloads the game swiftly. Well, I noticed that when trying to adjust the quality, a click on the slider actually causes the game to reload, and therefore you cannot move the slider.

**Comment by Quavelen** [comment]
Yup, I can confirm this using "OptiFine 1.14.3 HD U F1". Specifically, the Mipmap levels, type, and AF sliders.

**Comment by sp614x** [comment]
You can still use the sliders. 
Just click where the slider should be, don't try to drag it.

**Comment by LapisWolfMC** [comment]
Ah I see, that makes sense, however I still would like to see the sliders actually be able to slide. If it’s possible to do that, please do it!
